grammar.js

```diff
@@ -396,13 +398,23 @@ module.exports = grammar({
       field('body', $._suite)
     ),
 
+    _parenthesized_list_splat: $ => seq(
+      '(',
+      choice(
+        $._parenthesized_list_splat,
+        $.list_splat,
+      ),
+      ')',
+    ),
+
     argument_list: $ => seq(
       '(',
       optional(commaSep1(
         choice(
           $._expression,
           $.list_splat,
           $.dictionary_splat,
+          $._parenthesized_list_splat,
```

Comment:
I think it would be good to have this be a visible node, otherwise the `argument_list` will have three distinct children: `(`, `list_splat`, and `)`, with no indication that they represent a single argument.

Let's make `parenthesized_list_splat` visible, but alias it to `parenthesized_expression`, since it has the same structure as that node:

```js
alias($.parenthesized_list_splat, $.parenthesized_expression),
```